accessibility.md

```diff
@@ -0,0 +1,81 @@
+# Accessibility
+
+## Accessibility statement for DfE Architecture
+
+This accessibility statement applies to DfE Architecture at [https://dfe-digital.github.io/architecture/](https://dfe-digital.github.io/architecture/).
+
+This website is run by the architecture community at the Department for Education (DfE). We want as many people as possible to be able to use this website. For example, that means you should be able to:
+
++ change colours, contrast levels and fonts
++ zoom in up to 300% without problems
++ navigate most of the website using just a keyboard
++ navigate most of the website using speech recognition software
++ listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+
+We’ve also made the website text as simple as possible to understand.
+
+[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
+
+### How accessible this website is
+
+Currently the website should be fully accessible and there are no linked images or documents on the pages which could cause accessibility issues.
+
+### What to do if you cannot access parts of this website
+
+If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille, contact [architecture.services@education.gov.uk](mailto:architecture.services@education.gov.uk) with details of your request.
+
+We’ll aim to reply within 3 working days.
+
+### Reporting accessibility problems with this website
+
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, email [architecture.services@education.gov.uk](mailto:architecture.services@education.gov.uk).
+
+### Enforcement procedure
+
+The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, [contact the Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/).
+
+## Technical information about this website’s accessibility
+
+DfE is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+
+## Compliance status
+
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard, due to the non-compliances listed below.
+
+### Non-accessible content
+
+The following pages contain PNG images which are not fully explained in the text:
+
+- [https://dfe-digital.github.io/architecture/standards/common-definitions/#common-definitions](https://dfe-digital.github.io/architecture/standards/common-definitions/#common-definitions)
```

Comment:
Will try and fix these things asap

Reply:
@luke-slowen they're not pressing, GDS also have outstanding ones they haven't resolved yet: https://gds-way.cloudapps.digital/accessibility.html#non-accessible-content